src/commands/theme.js

```diff
@@ -89,7 +89,6 @@ export default function(program) {
           ];
 
           function renamePromiseFactory({oldFile, newFile, dir}) {
-            console.log(`  Renaming ${oldFile} to ${newFile}...`);
```

Comment:
Decided I didn't like the message after it said "you're done".

![](http://take.ms/0AiaFx)